pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-hadoop/pom.xml

```diff
@@ -119,6 +119,10 @@
           <groupId>org.apache.pinot</groupId>
           <artifactId>pinot-core</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Why do we need to exclude `hadoop-common` from `pinot-hadoop`? Seems a legit dependency?

Reply:
We are not gonna exclude all the `hadoop-common` from `pinot-hadoop`. `hadoop-client` keeps the dependency of `hadoop-common`:
```
[INFO] -------------------< org.apache.pinot:pinot-hadoop >--------------------
[INFO] Building Pinot Hadoop 0.6.0-SNAPSHOT                             [35/46]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ pinot-hadoop ---
[INFO] org.apache.pinot:pinot-hadoop:jar:0.6.0-SNAPSHOT
[INFO] \- org.apache.hadoop:hadoop-client:jar:2.7.0:provided
[INFO]    \- org.apache.hadoop:hadoop-common:jar:2.7.0:provided (scope managed from compile)
```